lib/Doctrine/bootstrap_doctrine_TEMPLATE.php

```diff
@@ -82,7 +80,10 @@
     /////////////////////////////////////////////////////////////////////////////////////////////
 
 
+    $evm = new EventManager();
```

Comment:
This can probably be moved to the top and the oracle specific line can be moved back to the oracle specific block

Reply:
Agreed - this should be done before merging

Reply:
this has been done and tested